features/searchable-dropdown.js

```diff
@@ -0,0 +1,110 @@
+// ==UserScript==
```

Comment:
Reminder that this script is over three years old (https://github.com/forkphorus/cat-plus/blob/master/searchable-dropdowns.user.js) and contains a massive amount of bugs. The equivalent addon in Scratch Addons (which was also originally based on this script) has received many significant fixes and features that are not present here.

Reply:
Alright, thank you very much! I will ask the beta testers to look out for bugs (which so far they haven't found any).

Reply:
Here's a few from memory to get you started:

 - can't press enter to select an item
 - arrow keys don't work properly
 - items are not sorted in any way
 - hovering over an item steals focus from the search bar
 - if I remember correctly, it interferes with angle inputs and maybe color ones too
 - the size of the dropdown can change as you search
 - ...

(The Scratch Addons version does not have any of these issues)